### ISSUE TITLE
Request to allow for alternate output emissions units

### ISSUE THREAD

**Issue by mkmahajan** [Energy-Innovation/eps-us#222]
Our China partners pointed out that it's more conventional in China to use 10^4 (wan) and 10^8 (yi) metric tonnes rather than MMT. We could easily allow for this by adding in an output unit conversion factor along with our other web-app files. 

This seems straightforward enough - if it sounds like a good idea, I can tag it as 3.3.2 and get it set up this week before our Friday programming session.

**Comment by jrissman** [comment]
Yes, we could allow the mass output units to be customized.  Note that we have two different mass output units, million metric tons for some pollutants (like CO2) and thousand metric tons for other pollutants (like PM), so we would need two different output conversion factors, for a "large" and a "small" mass output unit.  If you want to set up mass output units with conversion factors specified via a new input data variable, mimicking how the other conversion factors are set up, feel free to go ahead. :-)
